release-schema.json

```diff
@@ -30,6 +30,27 @@
             "null"
           ],
           "minLength": 1
+        },
+        "minimumRenewal": {
+          "title": "Minimum renewal",
+          "description": "The minimum number of renewals.",
+          "type": [
+            "number",
+            "null"
+          ]
+        },
+        "maximumRenewal": {
+          "title": "Maximum renewal",
+          "description": "The maximum number of renewals. If there is no maximum, set to 1e9999 (which parses to infinity).",
+          "type": [
+            "number",
+            "null"
+          ]
+        },
+        "renewalPeriod": {
```

Comment:
Should just be `period`, since it's already in scope of `Renewal` object.

Reply:
Done